### PR TITLE
Module names in quickfix && filter support

### PR DESCRIPTION
**Pull request by coot**
One PR instead of #2322 and #1757

### CODE REVIEW

File: runtime/doc/various.txt [hunk 1]
```diff
@@ -543,9 +543,19 @@ N  *+X11*		Unix only: can restore window title |X11|
 			bar character.
 
 			The pattern is matched against the relevant part of
-			the output, not necessarily the whole line. Only some
-			commands support filtering, try it out to check if it
-			works.
+			the output, not necessarily the whole line. The
+			following commands support filtering:
```
Comment (k-takata) [inline runtime/doc/various.txt:547]: I think `:map`, `:number` (same as `:#`), `:print` and `:z` also support filtering.

Comment (coot) [reply runtime/doc/various.txt:547]: Thanks I'll update the docs.

**Comment by brammool** [comment]
Marcin Szamotulski wrote:

> One PR instead of #2322 and #1757 

Sorry, that doesn't work.  One pull request per change please.
Commits can't be separated out easily, can't get the diff for each of
them.

-- 
An easy way to determine if you have enough teamwork to be doomed is simply to
measure how long it takes from the time you decide to go to lunch together
until the time you actually eat.
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by coot** [comment]
I see, then the only way is to reopen #1757 and make another PR after #1757 will get merged.  Sorry for the PR juggling.

**Pull request closed without merging**